### ISSUE TITLE
Use Ankit's research as mantik showcase

### ISSUE THREAD

**Issue by thomose** [mantik-ai/tutorials#6]
## Problem to solve

As mantik user I want to have a gallery of projects showcasing the platform, so that I can get an immediate idea of the platform's capabilities.

## Details

This story is only workable iff permission was granted in #5.

## Proposal

 - [x] Collect code dependencies in a `requirements.txt` file
 - [x] Create Dockerfile or apptainer recipe to install the project into a container
 - [x] Write entrypoint scripts and the corresponding `mlproject` file
 - [x] Include mlflow tracking to mantik into the scripts
 - [x] Create an experiment on `cloud.mantik.ai`
 - [x] Run the project on JUWELS and track to `cloud.mantik.ai`
 - [x] Write a README with a project description and citations of papers etc.

## Acceptance

 - [x] Project can be built and run
 - [x] Results are tracked to mantik
 - [x] README contains links to publications, original repo (if possible) and explicitly names Ankit as original author of the code

**Comment by Angie25** [comment]
General information
- Project's code is available in gitlab.
- An experimental dataset is available for running the code.
- The respective paper is under review. Discussion if the code can be published or not. There is no restriction from Ankit's side. 

Progress with the integration of the code into the platform
- The experiments would consist in pretraining and training.
- Experiments with tracking enabled can be run locally.

**Comment by Angie25** [comment]
README sample

Integrating the Seasonal Contrast project into the AI-platform.

This project is focused on atmospheric transformation used to generate views for contrastive self-supervised learning on multi-spectralimages. Here, the experiments are benchmarked on the SeCo algorithm. By using atmospheric transformation as an alternative to color jittering, all the experiments conducted by authors of SeCo can be extended to multiple channels instead of RGB. The pre-trained models trained using atmospheric transformation shows higher accuracy over the two downstream tasks i.e. EuroSAT and Bigearthnet when compared with one trained with color jittering and grayscaling. The experiments run on both Juwels Booster and Juwels Cluster. Through the AI-platform it is possible to manage the pretraining and the training workflows and also to track the experiments and their results.

Related papers: Atmospheric Correction for Contrastive Self-supervised Learning of Multi-spectral Images (planned to be submit this year: 2022)
Original author of the code: Ankit Patnala
Gitlab link of the repository: https://gitlab.jsc.fz-juelich.de/patnala1/seasonal-contrast/-/tree/main
